binary.go

```diff
@@ -67,12 +91,18 @@ func (bs *Binaries) Add(c []byte) *Binary {
 
 // GetContentBytes returns a bytes slice containing content of a binary
 func (b Binary) GetContentBytes() ([]byte, error) {
-	// Check for base64 content (KDBX 3.1), if it fail try with KDBX 4
-	decoded := make([]byte, base64.StdEncoding.DecodedLen(len(b.Content)))
-	_, err := base64.StdEncoding.Decode(decoded, b.Content)
-	if err != nil {
-		// KDBX 4 doesn't encode it
+	var decoded []byte
+
+	// KDBX 3.1 content is base64 encoded, KDBX 4 content is not
+	if b.isKDBX4 {
```

Comment:
I guess this flag is not necessary, this method was working fine for both kdbx3 and kdbx4.

Reply:
yeah, kind of not necessary - it would just be more consistent

Reply:
Adapted